app/components/forms/wizard/sponsors-step.js

```diff
@@ -23,39 +23,39 @@ export default Component.extend(FormMixin, {
     };
   },
 
-  sponsors: computed('data.event.sponsors.@each.isDeleted', function() {
-    return this.get('data.event.sponsors').filterBy('isDeleted', false);
+  sponsors: computed('data.sponsors.@each.isDeleted', function() {
+    return this.get('data.sponsors').filterBy('isDeleted', false);
   }),
 
   actions: {
     addSponsor() {
-      this.get('data.event.sponsors').addObject(this.store.createRecord('sponsor'));
+      this.get('data.sponsors').addObject(this.store.createRecord('sponsor'));
     },
     removeSponsor(sponsor) {
-      sponsor.deleteRecord();
+      sponsor.destroyRecord();
```

Comment:
@niranjan94 @CosmicCoder96 I would suggest to do that as a separate issue & get the event creation working first. What do you guys suggest?